source/_posts/2017-01-21-home-assistant-governance.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Home Assistant Governance"
+description: "Protecting Home Assistant for all, now and into the future"
+date: 2017-01-21 15:05:00 -0800
+date_formatted: "January 21, 2017"
+author: Paulus Schoutsen
+author_twitter: balloob
+comments: true
+categories: Organisation
+---
+
+The Home Assistant project and community has seen enormous growth in the last three years. Hundreds, if not thousands, of volunteers have and are working tirelessly every day on giving you all the awesomeness that you see today. That's not just the code but also all the related projects, documentation, examples, release blog posts and help in our forums and chat. And this is something that we want to keep safe and functional, even as we grow.
```

Comment:
`Hundreds, if not thousands, of volunteers have and are working tirelessly every day on giving you all the awesomeness that you see today.`

Don't change tense in the middle of a paragraph and especially not in the middle of a sentence. The first sentence is present perfect tense, the rest of the paragraph should stay in present tense.

`Many volunteers work tirelessly everyday to give you all the awesomeness that you see today.`